src/KingSystem/System/QuestMgr.cpp

```diff
@@ -0,0 +1,17 @@
+
+#include "KingSystem/System/QuestMgr.h"
+
+namespace ksys {
+
+SEAD_SINGLETON_DISPOSER_IMPL(QuestMgr);
```

Comment:
```suggestion
SEAD_SINGLETON_DISPOSER_IMPL(QuestMgr)
```

(the macro defines several functions, so having a semicolon at the end does nothing but produce warnings in some IDEs)